src/get-node-text.js

```diff
@@ -1,15 +1,11 @@
+const TEXT_NODE = 3
```

Comment:
Could we add a link to MDN docs that explain that this is `window.Node.TEXT_NODE` and the value is `3` (per the spec). Otherwise this is surprising.

Reply:
Good idea, updated the pr 😄